manifests/n/NetEase/identityv/1.0.0.1/netease.identityv.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=NVS2.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: netease.identityv
+PackageVersion: 1.0.0.1
+PackageLocale: en-US
+Publisher: Hong Kong NetEase Interactive Entertainment Limited
```

```suggestion
Publisher: Netease, Inc.
```
This needs to match Control Panel.